SamplesApp fails to build on Windows due to GLCanvas samples

### Current behavior

```
D:\Work\uno3\src\SamplesApp\UITests.Shared\Windows_UI_Composition\SimpleTriangleGlCanvasElement.cs(13,5,13,20): error CS0012: The type 'Grid' is defined in an assembly that is not referenced. You must add a reference to assembly 'Uno.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null'.
```

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Renderer

- [ ] Skia
- [ ] Native

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

--- a/src/AddIns/Uno.WinUI.Graphics3DGL/Uno.WinUI.Graphics3DGL.csproj
+++ b/src/AddIns/Uno.WinUI.Graphics3DGL/Uno.WinUI.Graphics3DGL.csproj
@@ -10,7 +10,7 @@
 		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 		
 		<!-- Required as the project name does not end with `.Skia` when building for netcoremobile targets -->
-		<UnoDisableTargetFrameworkPlatformOverride>true</UnoDisableTargetFrameworkPlatformOverride>
+		<UnoDisableTargetFrameworkPlatformOverride Condition="!$(UnoTargetFrameworkOverride.Contains('windows10'))">true</UnoDisableTargetFrameworkPlatformOverride>
     </PropertyGroup>
 
 	<Import Project="../../targetframework-override.props" />

